frontend/packages/dev-console/src/components/import/devfile/DevfileImportForm.tsx

```diff
@@ -45,26 +45,27 @@ const DevfileImportForm: React.FC<FormikProps<FormikValues> & DevfileImportFormP
             {devfileParseError}
           </Alert>
         )}
-        <FormSection>
-          {formType === 'sample' ? (
+        {formType === 'sample' ? (
+          <FormSection>
             <InputField
               type={TextInputTypes.text}
               name="git.url"
               label={t('devconsole~Git repo URL')}
               data-test-id="git-form-input-url"
               isDisabled
             />
-          ) : (
-            <GitSection
-              buildStrategy="Devfile"
-              builderImages={builderImages}
-              defaultSample={{
-                url: gitRepoUrl || 'https://github.com/redhat-developer/devfile-sample',
-              }}
-            />
-          )}
-          {selectedSample && <DevfileSampleInfo devfileSample={selectedSample} />}
-        </FormSection>
+          </FormSection>
+        ) : (
+          <GitSection
+            buildStrategy="Devfile"
+            builderImages={builderImages}
+            defaultSample={{
+              url: gitRepoUrl || 'https://github.com/redhat-developer/devfile-sample',
+            }}
+          />
+        )}
+        {selectedSample && <DevfileSampleInfo devfileSample={selectedSample} />}
```

Comment:
@debsmita1 since you are removing `DevfileSampleInfo` from `FormSection` as well can you confirm that no styles are breaking for  `DevfileSampleInfo`?

Reply:
![Peek 2021-06-16 00-41](https://user-images.githubusercontent.com/22490998/122110269-f85a6c80-ce3b-11eb-8bdd-dbb22dc92e82.gif)